modin/experimental/core/storage_formats/pandas/native_query_compiler.py

```diff
@@ -0,0 +1,1283 @@
+        row_numeric_index = list(row_numeric_index)
+    if not isinstance(col_numeric_index, slice):
+        col_numeric_index = list(col_numeric_index)
+    if not is_scalar(item):
+        broadcasted_items, _ = broadcast_item(
+            df,
+            row_numeric_index,
+            col_numeric_index,
+            item,
+            need_columns_reindex=need_columns_reindex,
+        )
+    else:
+        broadcasted_items = item
+
+    if isinstance(df.iloc[row_numeric_index, col_numeric_index], pandas.Series):
+        broadcasted_items = broadcasted_items.squeeze()
+    df.iloc[row_numeric_index, col_numeric_index] = broadcasted_items
+    return df
+
+
+def _setitem(df, axis, key, value):  # noqa: GL08
+    if is_scalar(key) and isinstance(value, pandas.DataFrame):
+        value = value.squeeze()
+    if not axis:
+        df[key] = value
+    else:
+        df.loc[key] = value
+    return df
+
+
+def _delitem(df, key):  # noqa: GL08
+    return df.drop(columns=[key])
+
+
+def _get_dummies(df, columns, **kwargs):  # noqa: GL08
+    return pandas.get_dummies(df, columns=columns, **kwargs)
+
+
+def _register_default_pandas(
+    func,
+    is_series=False,
+    squeeze_series=False,
+    squeeze_args=False,
+    squeeze_kwargs=False,
+    return_modin=True,
+    in_place=False,
+    df_copy=False,
+    filter_kwargs=[],
+):
+    """
+    Build function that apply specified method of the passed frame.
+
+    Parameters
+    ----------
+    func : callable
+        Function to apply.
+    is_series : bool, default: False
+        If True, the passed frame will always be squeezed to a series.
+    squeeze_series : bool, default: False
+        If True, the passed frame will always be squeezed to a series if there is a single column named "__reduced__".
+    squeeze_args : bool, default: False
+        If True, all passed arguments will be squeezed.
+    squeeze_kwargs : bool, default: False
+        If True, all passed key word arguments will be squeezed.
+    return_modin : bool, default: True
+        If True, the result will always try to convert to DataFrame or Series.
+    in_place : bool, default: False
+        If True, the specified function will be applied on the passed frame in place.
+    df_copy : bool, default: False
+        If True, the specified function will be applied to a copy of the passed frame.
+    filter_kwargs : list, default: []
+        List of key word argument names to remove.
+
+    Returns
+    -------
+    callable(pandas.DataFrame, *args, **kwargs) -> pandas.DataFrame
+        Function to be applied to the frame.
+    """
+
+    def caller(query_compiler, *args, **kwargs):
+        df = query_compiler._modin_frame
+        if df_copy:
+            df = df.copy()
+        if is_series:
+            df = df.squeeze(axis=1)
+        exclude_names = [
+            # "broadcast",
+            "fold_axis",
+            # "squeeze_self",
+            # "squeeze_value",
```

Comment:
```suggestion
            "fold_axis",
```
Should these be removed?